specification/cognitiveservices/data-plane/ComputerVision/stable/v2.0/Ocr.json

```diff
@@ -227,27 +227,33 @@
       }
     },
     "ComputerVisionError": {
-      "type": "object",
+      "description": "Details about the API request error.",
       "required": [
         "code",
         "message"
       ],
+      "type": "object",
       "properties": {
         "code": {
-          "type": "string",
           "description": "The error code.",
           "enum": [
-            "InvalidImageUrl",
             "InvalidImageFormat",
-            "InvalidImageSize",
-            "NotSupportedVisualFeature",
+            "UnsupportedMediaType",
+            "InvalidImageUrl",
+            "NotSupportedFeature",
             "NotSupportedImage",
-            "InvalidDetails",
-            "NotSupportedLanguage",
-            "BadArgument",
-            "FailedToProcess",
             "Timeout",
             "InternalServerError",
+            "InvalidImageSize",
+            "BadArgument",
+            "DetectFaceError",
+            "NotSupportedLanguage",
+            "InvalidThumbnailSize",
+            "InvalidDetails",
+            "InvalidModel",
+            "CancelledRequest",
+            "NotSupportedVisualFeature",
+            "FailedToProcess",
```

Comment:
This seems to be a breaking change for your API - current SDKs will not be able to handle these new enums. Only enums with "modelAsString" set to true are allowed to add new enum values.

Reply:
Can you help clarify?  If the service returns one of these new values in the JSON, aren't current clients screwed anyway?

Reply:
@jianghaolu Any comments?

Reply:
Yes That's what I mean by saying the rest API is going to break the clients.

Reply:
> aren't current clients screwed anyway

This doesn't justify breaking the clients after breaking the APIs. If this list is supposed to grow you should mark "modelAsString" to true in this PR. This PR is bringing breaking changes anyways (unfortunately) but there will be no more in the future.

Reply:
I changed it to true to prevent future breaking changes. Also [in here](https://github.com/Azure/azure-rest-api-specs/pull/4771/files#diff-adbdfc1efc9df7ca3f9f684ead4e71a5R1183).